cpp/arcticdb/version/test/test_stream_version_data.cpp

```diff
@@ -7,95 +7,102 @@ TEST(StreamVersionData, SpecificVersion) {
     using namespace arcticdb::pipelines;
 
     StreamVersionData stream_version_data;
-    VersionQuery query_1{SpecificVersionQuery{VersionId(12)}, false};
+    VersionQuery query_1{SpecificVersionQuery{VersionId(12)}};
     stream_version_data.react(query_1);
-    VersionQuery query_2{SpecificVersionQuery{VersionId(4)}, false};
+    VersionQuery query_2{SpecificVersionQuery{VersionId(4)}};
     stream_version_data.react(query_2);
     ASSERT_EQ(stream_version_data.count_, 2);
-    ASSERT_EQ(stream_version_data.load_param_.load_type_, LoadType::LOAD_DOWNTO);
-    ASSERT_EQ(stream_version_data.load_param_.load_until_version_, 4);
+    ASSERT_EQ(stream_version_data.load_strategy_.load_type_, LoadType::LOAD_DOWNTO);
```

Comment:
Should we lose the extra LOAD? as in LoadType::LATEST not LoadType::LOAD_LATEST? It's a bit smurf naming at the moment (not your fault) https://devcards.io/smurf-naming-convention